root/etc/e-smith/templates/openvpn-tunnel-client/40routes

```diff
@@ -1,10 +1,8 @@
 {
     use  NetAddr::IP;
     my $topology = $client->prop('Topology') || 'subnet';
-    if ($topology eq 'p2p') {
-        foreach (split(',',$client->prop('RemoteNetworks'))) {
-            my $net = NetAddr::IP->new($_);
-            $OUT .= sprintf("route %s %s\n",$net->addr(),$net->mask());
-        }
```

Comment:
Client configuration doesn't require any change:
- route is already pushed from the server when using subnet topology
- route should be explicitly added only when using p2p topology

Reply:
done